aws_lambda_builders/actions.py

```diff
@@ -32,6 +32,9 @@ class Purpose(object):
     # Action is compiling source code
     COMPILE_SOURCE = "COMPILE_SOURCE"
 
+    # Action is removing source code
+    REMOVE_SOURCE = "REMOVE SOURCEs"
```

Comment:
Indention looks off here. It matches `COMPILE_SOURCE` but not `COPY_SOURCE`.

This should be
`REMOVE_SOURCE = "REMOVE_SOURCE` to match the schema of the exiting Purposes